### DISCUSSION TITLE
Is Seagull able to improve Your tool?

### DISCUSSION THREAD

**Discussion by bw-development** [arriven/db1000n#468] [Ideas]
## Expected Behavior
Generate traffics that difficult to effectively block/recognize by gate fw/ids systems of target

About Seagull
http://gull.sourceforge.net/index.html

**Comment by arriven** [comment]
I don't think I would be able to import it directly (using CGO would make it pain in the ass to support different platforms) but based on my initial overview of it I feel like my packetgen package should allow similar functionality at some point and I could definitely draw inspiration from Seagull once I get to expand packetgen further

**Reply by bw-development** [reply]
Thank You for answer. 

I really hope You’ll find interesting for inspiration.
